homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -70,7 +70,9 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     for condition in monitored_conditions:
         if condition == "inverters":
             try:
-                inverters = await envoy_reader.inverters_production()
+                inverters = await hass.async_add_executor_job(
+                    envoy_reader.inverters_production
```

Comment:
`inverters_production` is a coroutine so this needs to be fixed in the library.

Reply:
https://github.com/jesserizzo/envoy_reader/blob/ee176190ebe95d98d6414deecf6d710438270cf9/envoy_reader/envoy_reader.py#L346-L363

Reply:
@jesserizzo Is this something you can take care of?

Reply:
Fix in https://github.com/jesserizzo/envoy_reader/pull/29